src/openpgp.js

```diff
@@ -93,6 +94,41 @@ export function destroyWorker() {
   asyncProxy = undefined;
 }
 
+//////////////////////////
+//                      //
+//   indutny/elliptic   //
+//                      //
+//////////////////////////
+
+
+let elliptic;  // instance of the indutny/elliptic
+
+/**
+ * Load elliptic by path or from node_modules
+ * @param {String} path relative path to elliptic browserified package
+ */
+export async function loadElliptic(path) {
+  if(typeof window !== 'undefined' && build.external_indutny_elliptic) {
+    // Fetch again if it fails, mainly to solve chrome bug "body stream has been lost and cannot be disturbed"
+    const ellipticPromise = util.dl({ filepath: path }).catch(() => util.dl({ filepath: path }));
```

Comment:
So you want to eliminate `path` option here? It would not work so simple as you could not load elliptic by static path stored in config. For example, you could look at `initWorker` function.

Reply:
I'm proposing that you call
```js
dl(config.elliptic_path, config.elliptic_fetch_options);
```
here. I know that's not what `initWorker` does, but `initWorker` is a function that's meant to always be called explicitly, whereas `loadElliptic` is meant to be called automatically by OpenPGP.js itself, so a config option makes sense.